can not run package.json scripts

i am not able to run my package.json scripts. Neither in standalone app nor in VSCode extension.

From VSCode extension, running a script results in this error

```
yarn run setup

/Users/alex/.yarn/bin/yarn.cmd: line 1: @echo: command not found
module.js:550
    throw err;
    ^

Error: Cannot find module '/Users/alex/repositories/go/src/gbc3/%~dp0\yarn.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3

Process failed 🐳
```

mention it tries to access the `yarn.cmd` but i am on OSX

From Standalone app the command is logged but no error is shown
```
yarn run setup
```

The structured output shows the message
```
Task failed. Please check terminal output for more details.
```

Running the commands does work when done manually from terminall
```
yarn run setup
npm run setup
```

I am on OSX 10.14.2 (18C54)
VSCode Version 1.31.1 (1.31.1)
Angular Console VSCode Version 7.3.1
Angula Console Electron Version 7.3.0 (7.3.0)

--- a/libs/server/src/lib/utils/file-utils.ts
+++ b/libs/server/src/lib/utils/file-utils.ts
@@ -80,9 +80,17 @@ export class FileUtils {
           command
         );
       }
-      if (existsSync(fullPath + '.exe')) {
+      if (
+        existsSync(fullPath + '.exe') &&
+        platform() === 'win32' &&
+        !this.isWsl()
+      ) {
         return fullPath + '.exe';
-      } else if (existsSync(fullPath + '.cmd')) {
+      } else if (
+        existsSync(fullPath + '.cmd') &&
+        platform() === 'win32' &&
+        !this.isWsl()
+      ) {
         return fullPath + '.cmd';
       } else if (existsSync(fullPath)) {
         return fullPath;

